python/cog/types.py

```diff
@@ -6,12 +6,30 @@
 import tempfile
 import urllib.parse
 import urllib.request
-from typing import Any, Dict, Iterator, List, Optional, TypeVar, Union
-
+import urllib.response
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Dict,
+    Iterator,
+    List,
+    Optional,
+    TypeVar,
+    Union,
+)
+
+import pydantic
 import requests
-from pydantic import Field, SecretStr
 from typing_extensions import NotRequired, TypedDict
 
+if pydantic.__version__.startswith("1."):
+    PYDANTIC_V2 = False
+else:
+    PYDANTIC_V2 = True
+    if TYPE_CHECKING:
+        pass  # type: ignore
```

Comment:
What does this do?

Reply:
`if TYPE_CHECKING`: pass` is a fingerprint left by Ruff's formatter, which automatically removes unused imports. That used to have some import, but it got removed. But instead of removing it, it got replaced with a `pass`. But we should kill these on sight.